src/LoggingConnector.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+namespace LeProxy\LeProxy;
+
+use React\Socket\Connector;
+use React\Promise\Promise;
+use React\Stream\WritableStreamInterface;
+use React\Socket\ConnectorInterface;
+use React\Socket\ConnectionInterface;
+use Clue\React\HttpProxy\ProxyConnector as HttpClient;
+use Clue\React\Socks\Client as SocksClient;
+
+/** @internal */
```

Comment:
I think everything is "internal" here :-) Perhaps add some (concise) documentation about what this class does instead?